static/js/publisher/release/components/releasesConfirmDetails/releaseRow.js

```diff
@@ -0,0 +1,44 @@
+import React, { Fragment } from "react";
+import PropTypes from "prop-types";
+
+const ReleaseRow = ({ type, revisionInfo, channel, progress, notes }) => (
+  <div className="p-release-details-row">
+    <span className="p-release-details-row__type">{type}</span>
+    <span className="p-release-details-row__info">
+      <span className="p-tooltip--btm-center">
+        <b>{revisionInfo.revision}</b> to{" "}
+        <span className="p-tooltip__message">
+          Version: <b>{revisionInfo.version}</b>
+        </span>{" "}
+        <b>{channel}</b> on <b>{revisionInfo.architectures.join(", ")}</b>
+      </span>
+    </span>
+    {progress && (
+      <Fragment>
+        <span className="p-release-details-row__join">to</span>
+        <span className="p-release-details-row__progress">{progress}</span>
+      </Fragment>
+    )}
+    {!progress && (
+      <Fragment>
+        <span />
```

Comment:
Why the empty spans?

Reply:
Ah yeah, I did that for the layout, but now you've pointed it out, I don't like it.